src/utils/formHelperFunctions.js

```diff
@@ -73,6 +82,96 @@ const formatAllStepFields = (steps, fieldValues) => {
     return formattedFields;
 };
 
+const generateRegexString = (rawPatternString) => {
```

Comment:
since the helper function file is getting pretty big, could we move string related utilities into something like a string.js?

Reply:
done.